doc/docusaurus/docs/delve-deeper/further-resources.md

```diff
@@ -4,18 +4,28 @@ sidebar_position: 40
+4. Manuel MT Chakravarty, James Chapman, Kenneth MacKenzie, Orestis Melkonian, Jann Müller, Michael Peyton Jones, Polina Vinogradova, and Philip Wadler. [Native Custom Tokens in the Extended UTXO Model](https://iohk.io/en/research/library/papers/native-custom-tokens-in-the-extended-utxo-model/). In *International Symposium on Leveraging Applications of Formal Methods*, 89–111. Springer, 2020.
 
-5. Manuel MT Chakravarty, Simon Thompson, and Philip Wadler. [Functional smart contracts on cardano](https://www.youtube.com/watch?v=MpWeg6Fg0t8).
+5. James Chapman, Roman Kireev, Chad Nester, and Philip Wadler. [System F in Agda, for fun and profit](https://iohk.io/en/research/library/papers/system-f-in-agda-for-fun-and-profit/). In *Mathematics of Program Construction*, 2019.
 
-6. IOHK. Formal specification of the Plutus Core language. Technical Report, IOHK, 2019. Available at [https://github.com/IntersectMBO/plutus](https://github.com/IntersectMBO/plutus).
+6. Michael Peyton Jones, Vasilis Gkoumas, Roman Kireev, Kenneth MacKenzie, Chad Nester, and Philip Wadler. [Unraveling recursion: compiling an IR with recursion to System F](https://iohk.io/en/research/library/papers/unraveling-recursion-compiling-an-ir-with-recursion-to-system-f/). In *Mathematics of Program Construction*, 2019.
 
-7. IOHK. Plutus platform technical report. Technical Report, IOHK, 2019. Available at [https://github.com/IntersectMBO/plutus](https://github.com/IntersectMBO/plutus).
+7. Jacco O.G. Krijnen, Manuel M.T. Chakravarty, Gabriele Keller, and Wouter Swierstra. [Translation certification for smart contracts](https://webspace.science.uu.nl/~swier004/publications/2024-scp.pdf). In *Science of Computer Programming*, 2024.
 
-8. Michael Peyton Jones and Jann Müller. [The plutus platform](https://www.youtube.com/watch?v=usMPt8KpBeI). 
+8. Polina Vinogradova and Orestis Melkonian. [Message-passing in the Extended UTxO Ledger Model](https://iohk.io/en/research/library/papers/message-passing-in-the-extended-utxo-ledger-model/). In *8th Workshop on Trusted Smart Contracts (WTSC)*, 2024.
+
+9. Polina Vinogradova, Orestis Melkonian, Philip Wadler, Manuel Chakravarty, Jacco Krijnen Michael Peyton Jones, James Chapman, and Tudor Ferariu. [Structured Contracts in the EUTxO Ledger Model](https://iohk.io/en/research/library/papers/structured-contracts-in-the-eutxo-ledger-model/). In *5th International Workshop on Formal Methods for Blockchains (FMBC)*, 2024.
+
+10. Manuel MT Chakravarty, Simon Thompson, and Philip Wadler. [Functional smart contracts on cardano](https://www.youtube.com/watch?v=MpWeg6Fg0t8).
+
+11. IOHK. Formal specification of the Plutus Core language. Technical Report, IOHK, 2019. Available at [https://github.com/IntersectMBO/plutus](https://github.com/IntersectMBO/plutus).
+
+12. IOHK. Plutus platform technical report. Technical Report, IOHK, 2019. Available at [https://github.com/IntersectMBO/plutus](https://github.com/IntersectMBO/plutus).
```

Comment:
This is definitely outdated ([here](https://input-output-rnd.slack.com/archives/G01EE4NQ9U0/p1720083189564909?thread_ts=1719917481.029889&cid=G01EE4NQ9U0)'s a Slack comment by Michael).